03/10/2021  12:03:1615393848 PM [ERROR] [root] App finished!
03/10/2021  12:03:1615393990 PM [ERROR] [root] App finished!
03/10/2021  12:03:1615394091 PM [ERROR] [root] App finished!
03/10/2021  12:03:1615394174 PM [ERROR] [root] App finished!
03/10/2021  12:03:1615394246 PM [ERROR] [root] App finished!
03/10/2021  12:03:1615394250 PM [ERROR] [root] Problems: could not open port  /dev/ttyUSB0 
03/10/2021  12:03:1615394264 PM [ERROR] [root] App finished!
03/10/2021  12:03:1615394941 PM [ERROR] [root] App finished!
03/10/2021  12:03:1615395100 PM [ERROR] [root] App finished!
03/10/2021  12:03:1615395160 PM [ERROR] [root] App finished!
03/10/2021  01:03:1615397259 PM [ERROR] [root] App finished!
03/10/2021  01:03:1615397337 PM [ERROR] [root] App finished!
03/10/2021  01:03:1615397408 PM [ERROR] [root] App finished!
03/10/2021  01:03:1615397623 PM [ERROR] [root] App finished!
03/10/2021  01:03:1615399116 PM [ERROR] [root] App finished!
03/10/2021  02:03:1615399278 PM [ERROR] [root] App finished!
03/10/2021  02:03:1615399521 PM [ERROR] [root] App finished!
03/10/2021  02:03:1615399792 PM [ERROR] [root] App finished!
03/10/2021  02:03:1615399913 PM [ERROR] [root] App finished!
03/10/2021  02:03:1615400365 PM [ERROR] [root] App finished!
03/10/2021  02:03:1615400505 PM [ERROR] [root] App finished!
03/10/2021  02:03:1615400604 PM [ERROR] [root] App finished!
03/10/2021  02:03:1615400623 PM [ERROR] [root] App finished!
03/10/2021  02:03:1615400702 PM [ERROR] [root] App finished!
03/10/2021  02:03:1615400752 PM [ERROR] [root] App finished!
03/10/2021  02:03:1615400780 PM [ERROR] [root] App finished!
03/10/2021  05:03:1615411722 PM [ERROR] [post] No internet
03/10/2021  05:03:1615411727 PM [ERROR] [root] App finished!
03/10/2021  05:03:1615411918 PM [ERROR] [post] Problems?
03/10/2021  05:03:1615411918 PM [ERROR] [post] No internet
03/10/2021  05:03:1615411922 PM [ERROR] [root] App finished!
03/10/2021  05:03:1615412000 PM [ERROR] [post] No internet
03/10/2021  05:03:1615412003 PM [ERROR] [root] App finished!
03/10/2021  05:03:1615412204 PM [ERROR] [post] Problems? (<class 'requests.exceptions.ConnectionError'>, ConnectionError(MaxRetryError("HTTPSConnectionPool(host='postman-echo.com', port=443): Max retries exceeded with url: /post (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fad5cb5f4e0>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),), <traceback object at 0x7fad5cb63908>)
03/10/2021  05:03:1615412204 PM [ERROR] [post] No internet
03/10/2021  05:03:1615412207 PM [ERROR] [root] App finished!
03/10/2021  05:03:1615412321 PM [ERROR] [post] Problems? (<class 'requests.exceptions.ConnectionError'>, ConnectionError(MaxRetryError("HTTPSConnectionPool(host='postman-echo.com', port=443): Max retries exceeded with url: /post (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f2bffc424a8>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),), <traceback object at 0x7f2bffc44908>)
03/10/2021  05:03:1615412321 PM [ERROR] [post] No internet
03/10/2021  05:03:1615412324 PM [ERROR] [root] App finished!
03/10/2021  05:03:1615412403 PM [ERROR] [post] No internet to send data
03/10/2021  05:03:1615412403 PM [ERROR] [post] No internet
03/10/2021  05:03:1615412406 PM [ERROR] [root] App finished!
03/10/2021  05:03:1615412529 PM [ERROR] [post] Connection Error!
03/10/2021  05:03:1615412529 PM [ERROR] [post] Post unsuccessful
03/10/2021  05:03:1615412547 PM [ERROR] [post] Connection Error!
03/10/2021  05:03:1615412547 PM [ERROR] [post] Post unsuccessful
03/10/2021  05:03:1615412550 PM [ERROR] [root] App finished!
03/10/2021  05:03:1615412839 PM [ERROR] [watchdog] Problems? (None, None, None)
03/10/2021  05:03:1615412839 PM [ERROR] [watchdog] Reseting script due to crashed
03/10/2021  05:03:1615413028 PM [ERROR] [root] App finished!
03/10/2021  05:03:1615413089 PM [ERROR] [root] Problems? (<class 'TypeError'>, TypeError('can only concatenate list (not "int") to list',), <traceback object at 0x7fc3dea40cc8>)
03/10/2021  05:03:1615413089 PM [ERROR] [root] Reseting script due to crashed
03/10/2021  05:03:1615413091 PM [ERROR] [root] Problems? (<class 'TypeError'>, TypeError('can only concatenate list (not "int") to list',), <traceback object at 0x7f54245c3cc8>)
03/10/2021  05:03:1615413091 PM [ERROR] [root] Reseting script due to crashed
03/10/2021  05:03:1615413280 PM [ERROR] [root] App finished!
03/10/2021  05:03:1615413314 PM [ERROR] [root] App Crashed... Restarting...
03/10/2021  05:03:1615413314 PM [ERROR] [root] Problems? (<class 'TypeError'>, TypeError("unsupported operand type(s) for +: 'int' and 'str'",), <traceback object at 0x7fe872efcd88>)
03/10/2021  05:03:1615413317 PM [ERROR] [root] App finished!
03/10/2021  05:03:1615413413 PM [ERROR] [central] Unexpected data lenght received [5, 0, 129, 3, 0, 255, 0, 120]
03/10/2021  05:03:1615413415 PM [ERROR] [root] App Crashed!
03/10/2021  05:03:1615413415 PM [ERROR] [root] Problems? (<class 'TypeError'>, TypeError("unsupported operand type(s) for +: 'int' and 'str'",), <traceback object at 0x7fa09ca62ec8>)
03/10/2021  05:03:1615413419 PM [ERROR] [root] App finished!
03/10/2021  05:03:1615413502 PM [ERROR] [central] Unexpected data lenght received [5, 0, 129, 3, 0, 255, 0, 120]
03/10/2021  05:03:1615413506 PM [ERROR] [root] App finished!
03/12/2021  03:03:1615576006 PM [ERROR] [root] App finished!
03/12/2021  03:03:1615576713 PM [ERROR] [root] Could not config LoRa
03/12/2021  03:03:1615577029 PM [ERROR] [root] Could not config LoRa
03/12/2021  03:03:1615577043 PM [ERROR] [root] Could not config LoRa
03/12/2021  03:03:1615577067 PM [ERROR] [watchdog] Problems? (None, None, None)
03/12/2021  03:03:1615577067 PM [ERROR] [watchdog] Reseting script due to timeout wdt
03/12/2021  03:03:1615577183 PM [INFO ] [root] App started
03/12/2021  03:03:1615577183 PM [INFO ] [central] This node ID: 0001 
03/12/2021  03:03:1615577183 PM [DEBUG] [central] Lora Serial Port: /dev/ttyUSB1 
03/12/2021  03:03:1615577183 PM [DEBUG] [central] Energy path: output/energy.json
03/12/2021  03:03:1615577184 PM [DEBUG] [root] Port  /dev/ttyUSB1  Available
03/12/2021  03:03:1615577184 PM [DEBUG] [central] Config frame: [1, 0, 1, 13, 165, 165, 108, 64, 18, 7, 0, 0, 1, 1, 0, 3, 0, 55]
03/12/2021  03:03:1615577189 PM [DEBUG] [central] Config Response: []
03/12/2021  03:03:1615577194 PM [INFO ] [central] Version LoRa: b''
03/12/2021  03:03:1615577194 PM [ERROR] [root] Could not config LoRa
03/12/2021  03:03:1615577200 PM [INFO ] [root] App started
03/12/2021  03:03:1615577200 PM [INFO ] [central] This node ID: 0001 
03/12/2021  03:03:1615577200 PM [DEBUG] [central] Lora Serial Port: /dev/ttyUSB1 
03/12/2021  03:03:1615577200 PM [DEBUG] [central] Energy path: output/energy.json
03/12/2021  03:03:1615577200 PM [DEBUG] [root] Port  /dev/ttyUSB1  Available
03/12/2021  03:03:1615577200 PM [DEBUG] [central] Config frame: [1, 0, 1, 13, 165, 165, 108, 64, 18, 7, 0, 0, 1, 1, 0, 3, 0, 55]
03/12/2021  03:03:1615577225 PM [INFO ] [root] App started
03/12/2021  03:03:1615577225 PM [INFO ] [central] This node ID: 0001 
03/12/2021  03:03:1615577225 PM [DEBUG] [central] Lora Serial Port: /dev/ttyUSB1 
03/12/2021  03:03:1615577225 PM [DEBUG] [central] Energy path: output/energy.json
03/12/2021  03:03:1615577225 PM [DEBUG] [root] Port  /dev/ttyUSB1  Available
03/12/2021  03:03:1615577225 PM [DEBUG] [central] Config frame: [1, 0, 1, 13, 165, 165, 108, 64, 18, 7, 0, 0, 1, 1, 0, 3, 0, 55]
03/12/2021  03:03:1615577230 PM [DEBUG] [central] Config Response: [1, 129, 8, 65, 150]
03/12/2021  03:03:1615577258 PM [INFO ] [root] App star
03/12/2021  03:03:1615577258 PM [INFO ] [central] This node ID: 0001 
03/12/2021  03:03:1615577258 PM [DEBUG] [central] Lora Serial Port: /dev/ttyUSB1 
03/12/2021  03:03:1615577258 PM [DEBUG] [central] Energy path: output/energy.json
03/12/2021  03:03:1615577259 PM [DEBUG] [root] Port  /dev/ttyUSB1  Available
03/12/2021  03:03:1615577259 PM [DEBUG] [central] Config frame: [1, 0, 1, 13, 165, 165, 108, 64, 18, 7, 0, 0, 1, 1, 0, 3, 0, 55]
03/12/2021  03:03:1615577282 PM [INFO ] [root] App started
03/12/2021  03:03:1615577282 PM [INFO ] [central] This node ID: 0001 
03/12/2021  03:03:1615577282 PM [DEBUG] [central] Lora Serial Port: /dev/ttyUSB1 
03/12/2021  03:03:1615577282 PM [DEBUG] [central] Energy path: output/energy.json
03/12/2021  03:03:1615577282 PM [DEBUG] [root] Port  /dev/ttyUSB1  Available
03/12/2021  03:03:1615577282 PM [DEBUG] [central] Config frame: [1, 0, 1, 13, 165, 165, 108, 64, 18, 7, 0, 0, 1, 1, 0, 3, 0, 55]
03/12/2021  03:03:1615577353 PM [DEBUG] [root] App started
03/12/2021  03:03:1615577353 PM [INFO ] [central] This node ID: 0001 
03/12/2021  03:03:1615577353 PM [DEBUG] [central] Lora Serial Port: /dev/ttyUSB1 
03/12/2021  03:03:1615577353 PM [DEBUG] [central] Energy path: output/energy.json
03/12/2021  03:03:1615577353 PM [DEBUG] [root] Port  /dev/ttyUSB1  Available
03/12/2021  03:03:1615577353 PM [DEBUG] [central] Config frame: [1, 0, 1, 13, 165, 165, 108, 64, 18, 7, 0, 0, 1, 1, 0, 3, 0, 55]
03/12/2021  03:03:1615577391 PM [ERROR] [root] Could not config LoRa
03/12/2021  03:03:1615577563 PM [ERROR] [root] Could not config LoRa
03/12/2021  03:03:1615577565 PM [ERROR] [central] Serial Port problem
03/12/2021  03:03:1615577581 PM [ERROR] [central] Serial Port problem
03/12/2021  03:03:1615577581 PM [ERROR] [central] Problems? (<class 'serial.serialutil.SerialException'>, SerialException('read failed: device reports readiness to read but returned no data (device disconnected or multiple access on port?)',), <traceback object at 0x7ff85bc05048>)
03/12/2021  03:03:1615577595 PM [ERROR] [central] Serial Port problem
03/12/2021  03:03:1615577595 PM [ERROR] [central] Problems? (<class 'serial.serialutil.SerialException'>, SerialException('read failed: device reports readiness to read but returned no data (device disconnected or multiple access on port?)',), <traceback object at 0x7fa758707048>)
03/12/2021  03:03:1615577619 PM [ERROR] [central] Serial Port problem
03/12/2021  03:03:1615577619 PM [ERROR] [central] Problems? (<class 'serial.serialutil.SerialException'>, SerialException('read failed: device reports readiness to read but returned no data (device disconnected or multiple access on port?)',), <traceback object at 0x7f32738373c8>)
03/12/2021  03:03:1615577648 PM [ERROR] [root] App finished!
03/12/2021  04:03:1615579450 PM [ERROR] [root] App finished!
03/19/2021  09:42:26 [ERROR] [root] Problems: could not open port  /dev/ttyUSB2 
03/19/2021  09:43:02 [ERROR] [central] Unexpected data lenght received [5, 0, 129, 3, 0, 1, 0, 134]
03/19/2021  09:43:17 [ERROR] [central] Unexpected data lenght received [5, 0, 129, 3, 0, 1, 0, 134]
03/19/2021  09:43:32 [ERROR] [central] Unexpected data lenght received [5, 0, 129, 3, 0, 1, 0, 134]
03/19/2021  09:43:47 [ERROR] [central] Unexpected data lenght received [5, 0, 129, 3, 0, 1, 0, 134]
03/19/2021  09:44:02 [ERROR] [central] Unexpected data lenght received [5, 0, 129, 3, 0, 1, 0, 134]
03/19/2021  09:44:17 [ERROR] [central] Unexpected data lenght received [5, 0, 129, 3, 0, 1, 0, 134]
03/19/2021  09:44:34 [ERROR] [central] Unexpected data lenght received [5, 0, 129, 3, 0, 1, 0, 134]
03/19/2021  09:44:37 [ERROR] [root] App finished!
03/19/2021  09:49:14 [ERROR] [central] Unexpected data lenght received [5, 0, 129, 3, 0, 1, 199, 65]
03/19/2021  09:49:15 [ERROR] [root] App finished!
03/19/2021  09:49:36 [ERROR] [root] App finished!
04/06/2021  08:58:25 [ERROR] [central] Unexpected data lenght received [5, 0, 129, 3, 0, 23, 199, 87]
04/06/2021  08:58:40 [ERROR] [central] Unexpected data lenght received [5, 0, 129, 3, 0, 23, 199, 87]
04/06/2021  08:58:55 [ERROR] [central] Unexpected data lenght received [5, 0, 129, 3, 0, 23, 199, 87]
04/06/2021  08:59:10 [ERROR] [central] Unexpected data lenght received [5, 0, 129, 3, 0, 23, 199, 87]
04/06/2021  08:59:25 [ERROR] [central] Unexpected data lenght received [5, 0, 129, 3, 0, 23, 199, 87]
04/06/2021  08:59:40 [ERROR] [central] Unexpected data lenght received [5, 0, 129, 3, 0, 23, 199, 87]
04/06/2021  08:59:56 [ERROR] [central] Unexpected data lenght received [5, 0, 129, 3, 0, 23, 199, 87]
04/06/2021  09:00:11 [ERROR] [central] Unexpected data lenght received [5, 0, 129, 3, 0, 23, 199, 87]
04/06/2021  09:00:26 [ERROR] [central] Unexpected data lenght received [5, 0, 129, 3, 0, 23, 199, 87]
04/06/2021  09:00:41 [ERROR] [central] Unexpected data lenght received [5, 0, 129, 3, 0, 23, 199, 87]
04/06/2021  09:00:56 [ERROR] [central] Unexpected data lenght received [5, 0, 129, 3, 0, 23, 199, 87]
04/06/2021  09:01:12 [ERROR] [central] Unexpected data lenght received [5, 0, 129, 3, 0, 23, 199, 87]
04/06/2021  09:01:27 [ERROR] [central] Unexpected data lenght received [5, 0, 129, 3, 0, 23, 199, 87]
04/06/2021  09:01:42 [ERROR] [central] Unexpected data lenght received [5, 0, 129, 3, 0, 23, 0, 144]
04/06/2021  09:02:25 [ERROR] [root] App finished!
04/07/2021  09:22:24 [ERROR] [central] Unexpected data lenght received [5, 0, 129, 3, 0, 23, 0, 144]
04/07/2021  09:23:47 [ERROR] [root] App finished!
04/14/2021  10:06:18 [ERROR] [root] App finished!
04/14/2021  10:11:24 [ERROR] [root] App finished!
04/14/2021  10:13:58 [ERROR] [root] App finished!
04/14/2021  10:19:31 [ERROR] [root] App finished!
04/14/2021  10:20:15 [ERROR] [root] App finished!
04/14/2021  10:20:46 [ERROR] [root] App finished!
04/14/2021  10:21:22 [ERROR] [root] App finished!
04/14/2021  10:21:53 [ERROR] [root] App finished!
04/14/2021  10:22:27 [ERROR] [root] App finished!
04/14/2021  10:23:03 [ERROR] [root] App finished!
04/14/2021  10:23:41 [ERROR] [central] Unexpected data lenght received [5, 0, 129, 3, 0, 23, 0, 144]
04/14/2021  10:24:26 [ERROR] [root] App finished!
04/14/2021  10:26:32 [ERROR] [root] App finished!
04/14/2021  10:29:30 [ERROR] [root] App finished!
04/14/2021  10:30:42 [ERROR] [root] App finished!
04/14/2021  10:31:06 [ERROR] [root] App finished!
04/14/2021  10:31:31 [ERROR] [central] Unexpected data lenght received [5, 0, 129, 3, 0, 23, 0, 144]
04/14/2021  10:31:36 [ERROR] [root] App finished!
04/14/2021  10:32:07 [ERROR] [root] App finished!
04/14/2021  10:36:26 [ERROR] [central] Unexpected data lenght received [5, 0, 129, 3, 0, 23, 0, 144]
04/14/2021  10:36:56 [ERROR] [root] App finished!
04/14/2021  10:38:28 [ERROR] [root] App finished!
04/14/2021  10:41:32 [ERROR] [root] App finished!
04/14/2021  10:45:32 [ERROR] [root] App finished!
04/14/2021  10:46:04 [ERROR] [root] App finished!
04/14/2021  10:46:21 [ERROR] [root] App finished!
04/14/2021  10:48:33 [ERROR] [central] Unexpected data lenght received [5, 0, 129, 3, 0, 23, 0, 144]
04/14/2021  10:48:51 [ERROR] [root] App finished!
04/14/2021  10:49:30 [ERROR] [root] App finished!
04/14/2021  10:51:32 [ERROR] [post] Problems? (<class 'requests.exceptions.MissingSchema'>, MissingSchema("Invalid URL 'facebook.com': No schema supplied. Perhaps you meant http://facebook.com?",), <traceback object at 0x7f934f7003c8>)
04/14/2021  10:51:32 [ERROR] [post] Post unsuccessful
04/14/2021  10:51:34 [ERROR] [root] App finished!
04/14/2021  10:52:58 [ERROR] [root] App finished!
04/14/2021  10:56:22 [ERROR] [root] App finished!
04/14/2021  11:03:51 [ERROR] [central] Unexpected data lenght received [5, 0, 129, 3, 0, 23, 0, 144]
04/14/2021  11:04:09 [ERROR] [root] App finished!
04/14/2021  11:05:20 [ERROR] [root] App finished!
04/14/2021  11:07:45 [ERROR] [root] App finished!
04/14/2021  11:08:28 [ERROR] [root] App finished!
04/14/2021  11:16:45 [ERROR] [root] App finished!
04/14/2021  11:17:20 [ERROR] [root] App finished!
04/14/2021  11:19:24 [ERROR] [central] Unexpected data lenght received [5, 0, 129, 3, 0, 23, 0, 144]
04/14/2021  11:19:39 [ERROR] [root] App finished!
04/14/2021  11:20:00 [ERROR] [root] App finished!
04/14/2021  11:20:25 [ERROR] [central] Unexpected data lenght received [5, 0, 129, 3, 0, 23, 0, 144]
04/14/2021  11:20:46 [ERROR] [root] App finished!
